app/controllers/catalog_controller.rb

```diff
@@ -267,15 +267,32 @@ class CatalogController < ApplicationController
     config.add_index_field 'format_main_ssim', label: 'Resource type', if: lambda { |context, *_args|
       context.feature_flags.add_resource_type_index_field?
     }
+
+    # Parker specific fields we do not wish to show in other exhibits
+    # editor_ssim, university_ssim, range_labels_ssim, and related_document_id_ssim.
+    config.add_index_field 'book_title_ssim', if: lambda { |context, *_args|
```

Comment:
@hudajkhan does this also need the label configured? It was set to "Book Title" below.

Reply:
Yes, I will add that